app/models/protocol.rb

```diff
@@ -550,40 +551,22 @@ def load_from_repository(source, current_user)
     save!
   end
 
-  def copy_to_repository(new_name, new_protocol_type, link_protocols, current_user)
-    clone = Protocol.new(
-      name: new_name,
-      description: description,
-      protocol_type: new_protocol_type,
-      added_by: current_user,
-      team: team
-    )
-    clone.published_on = Time.now if clone.in_repository_public?
+  def copy_to_repository(clone, current_user)
+    ts = DateTime.now
+    clone.team = team
+    clone.protocol_type = :in_repository_draft
+    clone.added_by = current_user
+    clone.record_timestamps = false
```

Comment:
I think we do not need this here anymore. Especially if we only create a new protocol template in the repository, without linking it to the task.  If we remove this then we do not next two lines  ( clone.created_at = ts, clone.updated_at = ts)